src/main/java/gregtech/common/metatileentities/storage/MetaTileEntityQuantumChest.java

```diff
@@ -264,6 +289,7 @@ public void readFromNBT(NBTTagCompound data) {
                 this.itemsStoredInside = data.getLong(NBT_ITEMCOUNT);
             }
         }
+        this.voiding = data.getBoolean("IsVoiding");
```

Comment:
This needs to check if the voiding tag is present in the NBT first, since it is a new tag that we are adding, and will not be present on existing machines.